jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -38,68 +38,42 @@
+ * - This implementation mimics the {@link com.jme3.anim.tween.AbstractTween}, but it delegates the interpolation method {@link Tween#interpolate(double)}
+ * to the {@link BlendableAction} class.
  *
- * Example of operation(Uses Abstract Factory Pattern Method(interpolate/doInterpolate)) :
- *
- * <li>#Check out #{@link BlendableAction} class.</li>
- * <li>#Custom Code : </li>
- * <pre class="prettyprint">
- *  public abstract class AbstractAction extends Action {
- *      private double length;
- *      private volatile float interpolationDelta = 0f;
- *      public ActionImpl(Tween... tweenActions){
- *          super(tweenActions);
- *      }
- *      public boolean interpolate(double t){
- *          //animation is out or in a reverse order.
- *          if(t < 0){
- *              return true;
- *          }
- *          //find where is the interpolation slider for transition.
- *          //interpolationDelta would be -plugged into the interpolateTransforms(transformI, transformII, delta)- as a shared value.
- *          interpolationDelta = t / length;
- *          doInterpolate(t);
- *          //interpolate as long as the interpolation slider haven't reached the full strength.
- *          return interpolationDelta < 1.0d;
- *      }
- *      public double getLength(){
- *           return length;
- *      }
- *      public void cloneFields(Cloner cloner, Object original) {
- *         //deep clone fields
- *      }
- *      public Action jmeClone() {
- *          //shallow clone
- *         try {
- *             return (Action) super.clone();
- *         } catch (CloneNotSupportedException exception) {
- *             throw new RuntimeException(exception);
- *         }
- *     }
- *      //override this factory method to do interpolation
- *      protected abstract void doInterpolate(double t);
- *  }
- * </pre>
  * <b>Created by Nehon.</b>
+ *
+ * @see BlendableAction
+ * @see BaseAction
  */
 public abstract class Action implements JmeCloneable, Tween {
 
     protected Action[] actions;
     private double length;
-    //the default speed is 1, which results in a normal tpf.
+    //the default speed is 1, which play the animation clips at their normal speed.
     private double speed = 1;
     private AnimationMask mask;
     private boolean forward = true;
 
     /**
-     * Instantiate an action object which encloses a couple of tween animation actions.
+     * Instantiate an action object which holds one or more tweens.
+     * If the tweens are {@link Action}, they would be added directly to {@link Action#actions}.
+     * If the tweens aren't {@link Action}, the Action class would delegate that to {@link BaseAction}, and {@link BaseAction} would try to extract the
+     * tween actions from the tweens arg and interpolate them.
+     * Notes :
+     * - If intentions are to make a holder class, then subclasses have to call this constructor, examples : {@link BlendableAction}, {@link BaseAction}.
```

Comment:
Those notes are to illustrate the behavior of the subclasses towards this constructor, and that's why not all subclasses of Action make a `super(tweens)` call.